src/utils/parseFunctionPriorityLabel.js

```diff
@@ -0,0 +1,4 @@
+export const parseFunctionPriorityLabel = id => {
```

Comment:
The function doesn't parse the "label" but rather generates it.
Please rename to `generateFunctionPriorityLabel`